05.ls/ls_command.rb

```diff
@@ -28,18 +29,17 @@ def parse_params
   params
 end
 
-def show_dot_file(params)
+def collect_files(params, path)
   flags = params[:a] ? File::FNM_DOTMATCH : 0
-  path = params[:path] || '.'
   Dir.glob('*', flags, base: path)
 end
 
-def sorted_files(gathered_files, params)
-  params[:r] ? gathered_files.reverse : gathered_files
+def sort_files(files, params)
+  params[:r] ? files.reverse : files
 end
 
-def make_detailed_file_info(files)
-  files.map do |filename|
+def make_detailed_file_info(sorted_files)
```

Comment:
mainメソッド内では並び替えられていない`files`と、並び替えられた`sorted_files`を区別する必要がありますが、区別の必要がなければ

```suggestion
def make_detailed_file_info(files)
```

でOKです。

加えて、このメソッドはfilesが並び替えられているかどうかに関係なく、渡された引数をひたすら表示するだけなので、`sorted`と配列の状態を限定する必要もありません。

参考 [ユーザーを見つけたら……Aさん「自転車に入れます」Bさん「人間に入れます」Cさん「ユーザーに入れます」 \| FBC](https://bootcamp.fjord.jp/pages/readable-variable-name)

> ただし、必ず一致させないといけないわけではないので、機械的に対応しないようにしてください。（メソッド定義側の引数名は多少抽象度が上がっても大丈夫なことが多い）